### DISCUSSION TITLE
react-dropzone upload file into formData ,but no get value

### DISCUSSION THREAD

**Discussion by anita2222** [react-dropzone/react-dropzone#1255] [Q&A]
I am do a file into formData ,but get the file is null ,I can't quest is where ,thanks !

I have a small code ,The formdata value cannot be obtained in the other method, and the transfer to the java backend file is also empty 。

const formData = new FormData();
const handleUpLoad=(event) =>{
    console.log("=="+event.target.files[0]);
    formData.append("files",event.target.files[0])
  }
 
 <div {...getRootProps()}>
    <input {...getInputProps()} />
    {
      isDragActive ?
        <p>Drop the files here ...</p> :
        <p>Drag 'n' drop some files here, or click to select files</p>
    }
 </div>

**Comment by rolandjitsu** [comment]
@anita2222 https://github.com/react-dropzone/react-dropzone/discussions/1367 may be helpful. Let me know if it does.
